### ISSUE TITLE
Cache Data fails to generate for Ubuntu 22.04

### ISSUE THREAD

**Issue by arshPratap** [openmainframeproject/software-discovery-tool#133]
It seems there is a bug in the system , while trying to setup the code on newer systems like **Ubuntu 22.04** where when attempting to setup discovery tool (instructions [here](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Installation.md)) it seems that the **cached_data.json** fails to assemble leading for the tool to not have anything to search for when testing the project locally. @pleia2 any thoughts  ?

**Comment by Princee215** [comment]
Yes, @arshPratap. When I tried to run it locally I was facing the same issue where **cached_data.json** is not generating on Ubuntu 22.04.

**Comment by arshPratap** [comment]
@pleia2 thanks for this..I just checked it and I realized the main issue is with IBMZ-container-registry.json file name incorrectly listed in the supported distros. As far as i remember, we had a PR #99 that included changing its name  and the same change needs to be reflected in the data repository. 
If possible let me add a quick PR there.

**Comment by arshPratap** [comment]
@pleia2 oh I think we need to update our git submodule for distro_data currently its pointing towards the commits made for the **may-branch**

**Comment by arshPratap** [comment]
@pleia2 @Princee215 I have added another PR with a slight update in the instructions for correctly initialising the git submodules.

**Comment by Princee215** [comment]
Thanks @arshPratap I tried and checked, and it is working well now.

**Comment by arshPratap** [comment]
@Princee215 thanks for the update ! Closing the issue
